docs/installation.md

```diff
@@ -9,37 +9,34 @@
 
 #Linux/Mac Automatic installation
 ### Easy installation
-1. Run setup.sh -e
-    This will create the needed encrypted.so file
-2. Run setup.sh -i
-    This will install the bot and all stuff that is needed to run it (follow the guide)
-3. Run setup.sh -c
-    This will make the config file needed, only basic stuff is changed here. If you want to edit more edit this file after: config/config.json
-4. Run run.sh
-    This will run the bot and will start leveling your pokemon go account.
+1. Clone the git: `git clone https://github.com/PokemonGoF/PokemonGo-Bot`
+2. Go into the new directory: `cd PokemonGo-Bot`
+3. Run `./setup.sh -i`  
+    This will install the bot and all stuff that is needed to run it (follow the steps in this process)
+4. Run `./run.sh`  
+    After you are done following it this will start your bot.
 
 ### To update
 1. Stop the bot if it's running. (use control + c twice to stop it)
-2. Run setup.sh -r
+2. Run `./setup.sh -r`  
     This will reset and makes sure you have no changes made to any code since it will overide it
-3. Run setup.sh -u
-    This will run git pull and will update to the new git update.
+3. Rerun the bot `./run.sh`
+
+note: we do not support windows at this time
 
 
 # Manual installation
 ### Protobuf 3 installation
 
 - OS X:  `brew update && brew install --devel protobuf`
 - Windows: Download protobuf 3.0: [here](https://github.com/google/protobuf/releases/download/v3.0.0-beta-4/protoc-3.0.0-beta-4-win32.zip) and unzip `bin/protoc.exe` into a folder in your PATH.
-- Linux: `apt-get install python-protobuf`
 
 ### Get encrypt.so (Windows part writing need fine tune)
-We don't have the copyright of encrypt.so, please grab from internet and build your self.Take the risk as your own.
-Example build sequence:
-Create a new separate folder some here
+Due to copywrite on the encrypt.so we are not directly hosting it. Please find a copy elsewhere on the internet and compile it yourself. We accept no responsibility should you encounter any problems with files you download elsewhere.
+
+Ensure you are in the PokemonGo-Bot main folder and run:
```

Comment:
to make it more noob proof so people don't do this in a random other folder that have noidea how linux work ;)